dev/io.openliberty.jakarta.jsonp.2.1_fat_tck/fat/src/io/openliberty/jakarta/jsonp/tck/JsonpTckLauncher.java

```diff
@@ -84,10 +84,25 @@ public void launchJsonpTCK() throws Exception {
                                            Collections.emptySet() //additional jars
         );
 
+        // Including jakarta.json-tck-tests and jakarta.json-tck-tests-pluggability together causes
+        // exceptions due to collisions, so created 2 separate profiles which are then
+        // run individually
+        additionalProps.put("run-tck-tests-pluggability", "true");
+        int result2 = MvnUtils.runTCKMvnCmd(
+                                            DONOTSTART, //server to run on
+                                            "io.openliberty.jakarta.jsonp.2.1_fat_tck", //bucket name
+                                            this.getClass() + ":launchJsonpTCK", //launching method
+                                            null, //suite file to run
+                                            additionalProps, //additional props
+                                            Collections.emptySet() //additional jars
+        );
```

Comment:
Can we pull this into its own test method and label the feature_name `jsonp-pluggability`?
This should result in two separate artifacts being persisted in our RTC builds which will simplify reporting.

Reply:
done